frontend/src/metabase/models/containers/NewModelOptions/NewModelOptions.tsx

```diff
@@ -44,6 +45,14 @@ const NewModelOptions = ({ location }: NewModelOptionsProps) => {
 
   const showMetabaseLinks = useSelector(getShowMetabaseLinks);
 
+  if (isFetching) {
+    return (
+      <Center style={{ flexGrow: 1, height: "100vh" }}>
+        <Loader size="lg" data-testid="loading-indicator" />
+      </Center>
+    );
```

Comment:
We should use the `DelayedLoadingSpinner`:
- the delay makes sense here
- it will automatically (and correctly) take available height and center itself
- we probably shouldn't use `Loader` directly - it'd be great if we had a few reusable `Loader`-based components that would take care of `data-testid` and a11y (which we lack as of now)

```suggestion
  return <DelayedLoadingSpinner />;
```